src/test/jsx.tsx

```diff
@@ -36,6 +36,16 @@ describe('snabbdom', function() {
       }));
     });
 
+    it('supports sel property shorthand', function() {
+      const vnode = <div sel="#id.foo.bar"></div>;
+
+      assert.equal(JSON.stringify(vnode), JSON.stringify({
+        sel: 'div#id.foo.bar',
+        data: {},
+        children: [],
+      }));
```

Comment:
There's `assert.deepStrictEqual`. Perhaps we should use that instead of JSON.

Reply:
I did try deepStrictEqual first, but deepStrictEqual looks at the undefined props as well and that fails. JSON.stringify is agnostic to undefined properties so our assert code can be smaller.

Also stringify gives a better error UX  because you see the full obj. deepStrictEquals only gives partial and wasn't as useful at first glance.

Reply:
Thank you fro the explanation. What's different about this test that it fails with `undefined` values? I'm at this moment converting all `deepEqual` to `deepStrictEqual` and nothing fails.

Reply:
Oh cool. Didn’t know it was fixed. Will fix then.

Reply:
Sorry but which `deepStrictEqual` do we use? In NodeJS, obviously, from standard `assert`. But in browser?

Reply:
@kuraga... good question. I don't know. It's a mystery.

Reply:
See https://github.com/snabbdom/snabbdom/pull/531#issuecomment-572177936, please.